### ISSUE TITLE
learning_rate becomes nan in rtho.py

### ISSUE THREAD

**Issue by stevenyu530** [awslabs/adatune#7]
Team,

I am trying to run your sample script ./bin/rtho.py, using the sample command in README.
`python bin/rtho.py --network vgg --dataset cifar_10 --optimizer sgd --momentum 0.9 --hyper-lr 1e-8`

After seeing val_cacurace increased in the first few epochs, learning_rate became nan. and training accuracy stuck at 10.0.
Sample output is attached at the end. 

Running baseline.py using the provided command runs fine. Best val_accuracy will reach good value.

Can you help to advise what is causing the learning rate nan issue in rtho.py ? 
Is is behaviour expected for the methods?

<img width="538" alt="Screenshot 2021-05-05 at 12 25 27" src="https://user-images.githubusercontent.com/5934114/117134069-00cf7a00-ad9d-11eb-9026-8d84f86ad2f8.png">

**Comment by orchidmajumder** [comment]
It will most likely be due to the other hyperparameters. I recommend using a lower value for those.
